pkg/build/api/v1/types.go

```diff
@@ -539,6 +539,10 @@ type BuildConfigStatus struct {
 type WebHookTrigger struct {
 	// Secret used to validate requests.
 	Secret string `json:"secret,omitempty"`
+
+	// AllowEnv determines whether the webhook can set environment variables; can only
```

Comment:
Public godoc for things that will end up in Godoc should follow "json" naming rules - "allowEnv determines whether ... to true for a generic web-hook"